pages/Addons.md

```diff
@@ -39,6 +39,7 @@ TranscEndence | Sfiguz | <details><summary>expand</summary>This Addon adds tools
 SFCalc | Seggan | <details><summary>expand</summary>This Addon is a continuation of the old SlimeCalculator by John000708. It provides a command to calculate the required amounts of base resources for any given Slimefun item.</details> | N/A | [GitHub](https://github.com/Seggan/SFCalc) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/SFCalc/master/)
 Tinkers (WIP) | Skizzles | <details><summary>expand</summary>An addon for Slimefun to add Tinkers Construct into Servers. This adds modifiers that can be crafted and added to Tools and Armor.</details> | N/A | [GitHub](https://github.com/Skizzles/Tinkers) | [Releases](https://github.com/Skizzles/Tinkers/releases)
 MoreTools (WIP) | LinoxGH | <details><summary>expand</summary>This addon adds more tools to Slimefun. These tools are meant to improve your gameplay experience, like Crescent Hammer(Wrench) and Cargo Copier.</details> | N/A | [GitHub](https://github.com/LinoxGH/MoreTools) | [Builds](https://thebusybiscuit.github.io/builds/LinoxGH/MoreTools/build/)
+FluffyMachines | NCBPFluffyBear | <details><summary>expand</summary>This addon adds an assortment of miscellaneous machines, generators, and tools!</details> | [Readme](https://github.com/NCBPFluffyBear/FluffyMachines/blob/master/README.md) | [Github](https://github.com/NCBPFluffyBear/FluffyMachines) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FluffyMachines/master/)
```

Comment:
That description is a bit too generic, try to give an example or two for your content.